js/Homework_js_3-4.js

```diff
@@ -0,0 +1,64 @@
+var oTest = {
+  /**
+   * Method that creates <h4> tag - the title of the test
+   */
+  createTitle: function () {
+    var oTestTitle = document.createElement('h4');
+    oTestTitle.innerHTML = 'Тест по программированию';
+    document.body.appendChild(oTestTitle);
+  },
+  /**
+   * Method that creates <ul> tags - the questions of the test
+   */
+  createQuestion: function () {
```

Comment:
Make questions as OL, don't put text between list and list items.

Reply:
done